src/Assets/subscribe.svg

```diff
@@ -0,0 +1,74 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0px" y="0px" viewBox="0 0 1080 1080" enable-background="new 0 0 1080 1080" xml:space="preserve">
```

Comment:
Why is this svg file added? I don't see it getting used anywhere. Please remove this if it's unnecessary.